### ISSUE TITLE
About pretrained model

### ISSUE THREAD

**Issue by ohosh** [huixiancheng/SENet#7]
Dear author,

Thanks for the sharing code.

 when I use the pretrained model about Kitti result you gave to run the datasets, the results can not reach the effect in your article. I don't know if there is a problem with the model I used, because there are multiple models in the file. I would like to ask which model you used to get the results in your article.

Thanks！

**Comment by huixiancheng** [comment]
Hi. The KITTI results only contain our training logs and results under 64x512 inputs. 
The model under the file named “512-594” you can get a test accuracy of 59.4. 
The file named “512+vaild-607” is the result after adding the validation set to the training and fine-tuning it, using this model test you should get a test set accuracy of 60.7 as reported in our paper.
Very sorry, pre-trained models and logs under larger inputs may not be considered for release

**Comment by cardwing** [comment]
@ohosh @huixiancheng I have trained the CENet which can achieve 65 mIoU on SemanticKITTI test set with 64x512 input resolution. The trained model will be put in https://github.com/cardwing/Codes-for-PVKD.

**Comment by cardwing** [comment]
More powerful range-image-based models built on the awesome CENet will also be put in that repo (70+ mIoU on SemanticKITTI test set).

**Comment by huixiancheng** [comment]
@cardwing Incredible. :open_mouth:  :scream:  :see_no_evil: That's awesome and amazing work. 
I think this will drive further development of the range-based methods. :thumbsup::thumbsup::thumbsup:
Looking forward to the release.

**Comment by cardwing** [comment]
The CENet with 64x512 input resolution has been uploaded to https://github.com/cardwing/Codes-for-PVKD. The reproduced performance (67.6% mIoU) is much higher than the reported value on SemanticKITTI test set.

**Comment by huixiancheng** [comment]
Glad to hear this. :clap: :thumbsup:
Modify README to point to that great work and Repo. :point_right:

**Comment by huishuai13** [comment]
> The CENet with 64x512 input resolution has been uploaded to https://github.com/cardwing/Codes-for-PVKD. The reproduced performance (67.6% mIoU) is much higher than the reported value on SemanticKITTI test set.

Is the CENet model with 64x512 in your repository trained with distillation?
